cluster/gce/util.sh

```diff
@@ -462,10 +462,32 @@ function detect-nodes() {
 #   KUBE_MASTER_IP
 function detect-master() {
   detect-project
-  KUBE_MASTER=${MASTER_NAME}
-  echo "Trying to find master named '${MASTER_NAME}'" >&2
+  if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
+    local server_name
+    for (( tp_num=1; tp_num<=${SCALEOUT_TP_COUNT}; tp_num++ ))
+    do
+      server_name="${INSTANCE_PREFIX}-tp-${num}-master"
```

Comment:
i saw sometime the culster_name is used. what is the purposes for cluster_name and instance_prefix variables ?

Reply:
Instance_prefix only has prefix, which is global same for kube-up or kubemark, for example for kube-up: perf1-1*1*100; kubemark: perf1-1*1*100-kubemark
Cluster_Name is cluster master instance specific,  example for kube-up: perf1-1*1*100-tp-1; kubemark: perf1-1*1*100-kubemark-tp-1
Master_Name is instance name, example for kube-up: perf1-1*1*100-tp-1-master; kubemark: perf1-1*1*100-kubemark-tp-1-master